### ISSUE TITLE
Find missing href attributes in rancherio/ui and replace them with `href="#"`

### ISSUE THREAD

**Issue by ndarilek** [rancher/rancher#1405]
Tried to file this issue there but the repository doesn't seem to support them. A few weeks back I submitted a PR adding `href="#"` to a link, which is necessary for accessibility. I received instructions on how to switch my UI to the version from master, but those instructions referenced a link that also seems to be missing an href. In particular, I think the account icon referenced in the getting started docs, which leads to the APIs and Keys page, is missing an href. At least, I can't find it using my screen reader and assume this to be the case. I'd look into submitting a PR, but by definition I can't find any affected links myself, and it might be wise to start making this fix yourselves to avoid it in the future.

Thanks.

**Comment by lvuch** [comment]
resolved in https://github.com/rancher/ui/pull/511

**Comment by lvuch** [comment]
resolved in https://github.com/rancher/ui/pull/511

**Comment by tfiduccia** [comment]
Version - master 3/8
Verified fixed
